test/case/ietf_routing/ospf_multiarea/Readme.adoc

```diff
@@ -1,10 +1,10 @@
 === OSPF with multiple areas
 ==== Description
-This test test alot of features inside OSPF using 3 areas (one NSSA area, with no summary)
+This test tests a lot of features inside OSPF using 3 areas (one NSSA area, with no summary)
```

Comment:
Maybe we could rephrase this a little bit if you like :
"This test evaluates various OSPF features across three areas (one NSSA area without summary) to ensure that route distribution is deterministic (based on cost). It also tests link failures using BFD, though BFD is not yet implemented in Infamy."

Reply:
I'll use most of your suggestions. Thx!